lib/interceptor/retry.js

```diff
@@ -1,19 +1,27 @@
 'use strict'
```

Comment:
Don't modify this.

Reply:
<img width="391" alt="Screenshot 2024-06-17 at 14 59 49" src="https://github.com/nodejs/undici/assets/37827216/ba2a9198-dd48-43af-a544-8208ceea1564">
<img width="391" alt="Screenshot 2024-06-17 at 14 59 49" src="https://github.com/nodejs/undici/assets/37827216/ba2a9198-dd48-43af-a544-8208ceea1564">

I wonder how I can find it

Reply:
On the `index.js` export it like we export other interceptors.

Reply:
did it here I hope it's the right solution https://github.com/nodejs/undici/pull/3331/commits/2195af1d6f2480f7a1d7f4474cfe1d38f08978c0